Formula/adios2.rb

```diff
@@ -0,0 +1,126 @@
+class Adios2 < Formula
+  desc "Next generation of ADIOS developed in the Exascale Computing Program"
+  homepage "https://adios2.readthedocs.io"
+  url "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+  sha256 "7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329"
+  head "https://github.com/ornladios/ADIOS2.git", :branch => "master"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "bzip2"
+  depends_on "c-blosc"
+  depends_on "catch2"
```

Comment:
@ax3l just curious about this dependency, adios2 doesn't explicitly depend on it.

Reply:
ups, good catch. I copied from my openPMD-api recipe. Will remove!